src/basic/random-util.c

```diff
@@ -257,8 +103,6 @@ void initialize_srand(void) {
 #if HAVE_SYS_AUXV_H
         const void *auxv;
```

Comment:
While this code is touched, I'd love to see this declaration be moved into the other `#if`.

Reply:
There's a statement in between the two, and afaict, systemd declares variables before statements.

Reply:
That restriction was removed some time ago.

Reply:
Oh. Maybe send another PR for that? I don't see how it's related to this, and it'd be nice to have this stand-alone revertable in case things don't go as planned.